### DISCUSSION TITLE
使用opencompass如何自定义数据集，并且自定义Evaluator进行评测指标计算，具体到底要编写哪些配置文件

### DISCUSSION THREAD

**Discussion by Liangzi-qyh** [open-compass/opencompass#1535] [Q&A]
使用opencompass如何自定义数据集，并且自定义Evaluator进行评测指标计算，具体到底要编写哪些配置文件

**Comment by bittersweet1999** [comment]
参见https://github.com/open-compass/opencompass/blob/main/docs/zh_cn/advanced_guides/custom_dataset.md
https://github.com/open-compass/opencompass/blob/main/docs/zh_cn/advanced_guides/new_dataset.md

**Comment by tonysy** [comment]
Hi,  here are some examples
- https://github.com/open-compass/opencompass/pull/1099/files#
- https://github.com/open-compass/opencompass/pull/1417
- https://github.com/open-compass/opencompass/pull/729/files

**Reply by Liangzi-qyh** [reply]
Thank you for your answer, I carefully study the examples you provided, and I will ask you again if I have questions
